rel/i18n/emqx_schema.hocon

```diff
@@ -1633,10 +1633,15 @@ This inevitably limits the overall throughput of the system.
 session_ds_heartbeat_interval.label:
 """Durable session heartbeat interval"""
 session_ds_heartbeat_interval.desc:
-"""Durable sessions back up their state on disc every heartbeat interval.
+"""The interval at which a node with durable sessions reports its liveness.
```

Comment:
Please rollback. Sessions still need to save their data while online.

Reply:
They do